resources/public/api/conf/1.0/testdata/fund-release.md

```diff
@@ -0,0 +1,446 @@
+                                                                              </td>
+                                                                              <td>
+                                                                              <pre class="code--block"> 
+  {
+    "eventDate": "2017-05-10",
+    "withdrawalAmount": 4000.00,
+    "conveyancerReference": "CR12345-6789",
+    "propertyDetails": {
+           "nameOrNumber": "1",
+           "postalCode": "AA11 1AA",
+     }
+  }   
+                                                                                </pre>
+                                                                                </td>
+                                                                                <td>
+                                                                                   <p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+                                                                                   <pre class="code--block">
+  {
+    "code": "INVESTOR_ACCOUNT_ALREADY_CANCELLED",
+    "message": "The LISA account has already been cancelled"
+  }                                               
+                                                                                 </pre>
+                                                                                 </td>
+                                                                                 </tr>   
+                                                                                     <tr>
+                                                                                         <td>
+                                                                                            <p>Account not open long enough</p>
+                                                                                            <p class="code--block">
+                                                                                            <strong>lisaManagerReferenceNumber:</strong><br>
+                                                                                            <a href="https://test-developer.service.hmrc.gov.uk/api-documentation/docs/api/service/lisa-api/1.0#testing-the-api">Use your test user profile</a><br>
+                                                                                            <br>
+                                                                                            <strong>accountId:</strong><br>4030000011
+                                                                                            </p>
+                                                                                            </td>
+                                                                                                 <td>
+                                                                                            <pre class="code--block"> 
+ {
+   "eventDate": "2017-12-10",
```

Comment:
Can this json be the same as other examples? The event date and conveyancerReference are different.